### ISSUE TITLE
Can't build and run after upgrade from Vaadin 17.0.11 to 18.0.1

### ISSUE THREAD

**Issue by Tonne-TM** [vaadin/flow#9554]
### Description of the bug / feature
Problems after updating from Vaadin 17.0.11 to 18.0.1

When starting in developement mode I get the following error:
```
2020-12-02 12:57:31.870 ERROR 22416 --- [nio-8080-exec-1] c.v.flow.server.frontend.FrontendUtils   : Cannot get the 'META-INF/VAADIN/index.html' from the classpath
2020-12-02 12:57:31.876 ERROR 22416 --- [nio-8080-exec-1] c.v.flow.server.DefaultErrorHandler      : 

java.io.UncheckedIOException: java.io.IOException: Failed to load content of './frontend/\/index.html'. It is required to have './frontend/\/index.html' file when using client side bootstrapping.
```

When building production image webpack throws the following error (starting Vaadin 18.0.x)
```
ERROR in ./styles/globalVariables.css
Module build failed (from ../node_modules/.pnpm/registry.npmjs.org/css-loader/4.2.1_webpack@4.42.0/node_modules/css-loader/dist/cjs.js):
Error: Can't resolve 'frontend/images/logo.svg' in 'D:\Workspace\webapp\frontend\styles'
```
### Minimal reproducible example
My frontend folder looks like this:
```
frontent
- src
- styles
- index.html
```

In `globalVariables.css` I have the following line:
```
--logoImage: url("frontend/images/logo.svg");
```

The logo is located in: `webapp\src\main\resources\static\images\logo.svg`

### Expected behavior
No problem when running/building image
### Actual behavior
Error when running/building image
### Versions:
    - Vaadin / Flow version: 18.0.1
    - Java version: 11
    - OS version: Windows 10
    - Spring Boot: 2.4.0

**Comment by pleku** [comment]
Thanks for the report - the untested workaround would be to add the following inside `webpack.config.js` before the line `module.exports = merge(flowDefaults,`:
```js
const cssLoaders = flowDefaults.module.rules.find(r => r.test && r.test.toString() == '/\\.css$/i');
const index = cssLoaders.use.indexOf('css-loader');
if (index !== -1) {
  const cssLoaderConfig = {
    loader: 'css-loader',
    options: { url: false }
  };
  cssLoaders.use.splice(index, 1, cssLoaderConfig);
}
```
we'll land a fix for this soon that makes css-loader touch only `node_modules` urls.

**Comment by pleku** [comment]
The odd part is, that while there is a regression in v18, I don't understand how the use case has worked other than "by accident" v17 as the URL in css for `url("frontend/images/logo.svg");` should not map in any way to `\src\main\resources\static\images\logo.svg` which is a standard static resources directory and the contents are available always from `images\logo.svg` (unless blocked by security configuration). So this is why the case to reproduce this would be interesting for us.

There is something touching the url and making it work (to be located) but "it is not right". So I think your should change the url in css to be `images/logo.svg` as the content is not inside the frontend folder.

**Comment by Tonne-TM** [comment]
First of all the suggestet fix of @pleku worked (thanks!). I was also confused my this path was working. But at it turned out the path I described was not the correct path.

I did a little more digging and discovered that the static folder contained a folder named `frontend` for some odd reason which caused all the trouble:
`webapp\src\main\resources\static\fontend\images\logo.svg`

So to reproduce the error you have to create this folder. Of course in my project I deleted this folder.

**Comment by pleku** [comment]
This has been fixed in 5.0.2 / 18.0.2
